docs-src/tic-tac.scrbl

```diff
@@ -0,0 +1,1437 @@
+#lang scribble/manual
+@(require "lib.rkt")
+
+@title[#:version reach-vers #:tag "top" #:style 'toc]{Pay-To-Play Tic-Tac-Toe Tutorial}
+
+
+This tutorial walks through the creation of a Pay-to-Play Tic-Tac-Toe decentralized application using Reach.
+Pay-To-Play Tic-Tac-Toe is a game very similar to regular Tic-Tac-Toe, except that each place on the board has a fee that is calculated by how many 'wins' can be made by utilizing chosen place.
+It contains everything you need to know to build and test this application and assumes no prior experience with blockchain development of any kind.
+
+To dive in, click to go to the @seclink["tic-tac-0"]{first step}!
+
+@local-table-of-contents[#:style 'immediate-only]
+
+
+@section[#:tag "tic-tac-0"]{Install and Initialize}
+@(mint-scope "tic-tac-0")
+
+Reach is designed to work on POSIX systems with @link["https://en.wikipedia.org/wiki/Make_(software)"]{make}, @link["https://www.docker.com/get-started"]{Docker}, and @link["https://docs.docker.com/compose/install/"]{Docker Compose} installed.
+The best way to install Docker on Mac and Windows is with @link["https://www.docker.com/products/docker-desktop"]{Docker Desktop}.
+
+@margin-note{You probably already have @exec{make} installed.
```

Comment:
Make........ @squidKid-deluxe you may just want to ctrl+F "make" and correct the capitalization so you don't have to resolve hundreds of comments.